src/less/toast-dialog/toast-dialog.less

```diff
@@ -65,8 +65,9 @@
 
 // inherits from .icon-btn
 button.toast-dialog__close {
-    align-self: center;
+    align-self: start;
     border: 0;
+    flex-shrink: 0;
```

Comment:
Is this needed? What are the implications if it's not included?

Reply:
If this isn't included the button compresses vertically and the icon shrinks to a small portion of its size. This is needed.